STATUS.md

```diff
@@ -0,0 +1,489 @@
+- [ ] `DescBomb.guidance`
+- [ ] `DescBomb.altMin`
+- [ ] `DescBomb.altMax`
+
+### Unit (inherits CoalitionObject)
+Primarily enhanced with `GRPC.exporters.unit`:
+- [ ] `isActive`
+- [ ] `getPlayerName`
+- [x] `getID`
+- [ ] `getNumber`
+- [ ] `getController`
+- [ ] `getGroup`
+- [x] `getCallsign`
+- [ ] `getLife`
+- [ ] `getLife0`
+- [ ] `getFuel`
+- [ ] `getAmmo`
+- [ ] `getSensors`
+- [ ] `hasSensors`
+- [ ] `getRadar`
+- [ ] `getDrawArgumentValue`
+- [ ] `getNearestCargos`
+- [ ] `enableEmission`
+- [ ] `getDescentCateogry`
+- [ ] (static) `getByName`
+- [ ] (static) `getDescByName`
+
+### Airbase (inherits CoalitionObject)
+- [ ] `getDesc`
+- [ ] `getCallsign`
+- [ ] `getUnit`
+- [ ] `getID`
+- [ ] `getParking`
+- [ ] `getRunways`
+- [ ] `getTechObjectPos`
+- [ ] `getRadioSilentMode`
+- [ ] `setRadioSilentMode`
+
+### StaticObject (inherits CoalitionObject)
+- [ ] `getID`
+- [ ] `getLife`
+- [ ] `getCargoDisplayName`
+- [ ] `getCargoWeight`
+- [ ] `getDrawArgumentValue`
+- [ ] (static) `getByName`
+- [ ] (static) `getDescByName`
+
+### Group
+- [ ] (static) `getByName`
+- [ ] `isExist`
+- [ ] `activate`
+- [ ] `destroy`
+- [ ] `getCategory`
+- [ ] `getCoalition`
+- [ ] `getName`
+- [ ] `getID`
+- [ ] `getUnit`
+- [ ] `getUnits`
+- [ ] `getSize`
+- [ ] `getController`
+- [ ] `enableEmission`
+
+### Controller
+- [ ] `setTask`
+- [ ] `resetTask`
+- [ ] `pushTask`
+- [ ] `popTask`
+- [ ] `hasTask`
+- [ ] `setCommand`
+- [ ] `setOption`
+- [ ] `setOnOff`
+- [ ] `knowTarget`
+- [ ] `isTargetDetected`
+- [ ] `getDetectedTargets`
+- [Main Tasks](https://wiki.hoggitworld.com/view/DCS_task_mission)
+  - [ ] `mission`
+  - [ ] `AttackGroup`
+  - [ ] `AttackUnit`
+  - [ ] `Bombing`
+  - [ ] `CarpetBombing`
+  - [ ] `AttackMapObject`
+  - [ ] `BombingRunway`
+  - [ ] `orbit`
+  - [ ] `refueling`
+  - [ ] `land`
+  - [ ] `follow`
+  - [ ] `followBigFormation`
+  - [ ] `escort`
+  - [ ] `Embarking`
+  - [ ] `fireAtPoint`
+  - [ ] `hold`
+  - [ ] `FAC_AttackGroup`
+  - [ ] `EmbarkToTransport`
+  - [ ] `DisembarkFromTransport`
+  - [ ] `CargoTransportation`
+  - [ ] `goToWaypoint`
+  - [ ] `groundEscort`
+- Enroute Tasks
+  - [ ] `engageTargets`
+  - [ ] `engageTargetsInZone`
+  - [ ] `engageGroup`
+  - [ ] `engageUnit`
+  - [ ] `awacs`
+  - [ ] `tanker`
+  - [ ] `ewr`
+  - [ ] `FAC_engageGroup`
+  - [ ] `FAC`
+- Commands
+  - [ ] `script`
+  - [ ] `setCallsign`
+  - [ ] `setFrequency`
+  - [ ] `switchWaypoint`
+  - [ ] `stopRoute`
+  - [ ] `switchAction`
+  - [ ] `setInvisible`
+  - [ ] `setImmortal`
+  - [ ] `activateBeacon`
+  - [ ] `deactivateBeacon`
+  - [ ] `eplrs`
+  - [ ] `start`
+  - [ ] `transmitMessage`
+  - [ ] `stopTransmission`
+  - [ ] `smoke_on_off`
+- [Options](https://wiki.hoggitworld.com/view/DCS_func_setOption)
+  - [ ] `ROE`
+  - [ ] `Reaction To Threat`
+  - [ ] `Radar Using`
+  - [ ] `Flare Using`
+  - [ ] `Formation`
+  - [ ] `RTB On Bingo`
+  - [ ] `silence`
+  - [ ] `Disperse on Attack`
+  - [ ] `Alarm State`
```

Comment:
This has been implemented: https://github.com/DCS-gRPC/rust-server/blob/status/protos/dcs/controller/v0/controller.proto#L8